sentry-anyhow/src/lib.rs

```diff
@@ -29,6 +29,16 @@
 //! }
 //! ```
 //!
+//! # Features
+//!
+//! The `backtrace` feature will enable the corresponding feature in anyhow and allow you to
+//! capture backtraces with your events.  It is enabled by default.
+//!
+//! # Resources
+//!
+//! - [Discord](https://discord.gg/ez5KZN7) server for project discussions.
+//! - Follow [@getsentry](https://twitter.com/getsentry) on Twitter for updates.
+//!
```

Comment:
This is added by cargo-readme via the template. Not sure this should be part of the rustdoc.